### ISSUE TITLE
Unify the use `preview_image_link` across Volto

### ISSUE THREAD

**Issue by sneridagh** [plone/volto#5630]
## Abstract

Currently, Plone/Volto use `preview_image` field in content types to store images as an attribute (NamedBlobImage) in the DX content type itself. This leads to duplication in case you want to reuse images across your content.

`preview_image_link` stores a RelationChoice pointer instead (in the server a resolveuid). 

## Motivation

Allow reusable images and open the door to a DAMS (Digital Asset Management System) in Plone.

## Assumptions

No preconditions.

## Proposal & Implementation

Change the type of the fields in the content types.

## Deliverables

A migration path to the new scenario from the old version.

## Risks

The migration involved.

**Comment by davisagli** [comment]
@sneridagh What is the specific proposal?

plone.volto already supports the preview_image_link behavior, but it is not enabled by default.

Is this a proposal to disable the preview_image behavior and enable preview_image_link by default?

We could pretty easily do that for new sites only, and thus avoid a migration (as long as we still support the old preview_image behavior when it is used)

If we want to get rid of preview_image entirely and always use preview_image_link, that would require a migration which creates a new Image content item for each image that is currently stored in an image field. That could take quite some time for a large site, and bloat the size of the catalog.

It can also be a complicated question how to handle translations when the images are separate content items.

**Comment by sneridagh** [comment]
This was motivated by the request @tisto did the other day. I wanted to put it on the table to start discussions. I know that the migration could be painful. I was wondering if we can have it in specific distributions. Also, to raise awareness of this option, since I realised that even some members of the Volto Team didn't know about it.

**Comment by davisagli** [comment]
@sneridagh Like I said, I'm not sure we need the migration. We could make the switch for new sites only.

**Comment by tisto** [comment]
@davisagli @sneridagh I agree with David. We only need this as an option for new sites so that integrators can choose. If we want to make this a default at some point is a different discussion. I would love to see an option that allows both uploading an image and re-using an existing image. However, this is a bit more complicated.

**Comment by sneridagh** [comment]
@tisto @davisagli ok, however, we have to make sure that it is supported everywhere thorough the Volto code base (also, I received feedback from some Volto Team Members that they did not know about this, so we will need to put some effort in documenting it properly).

OTOH, if we want at some point to integrate the DAMS in Volto we might want to reconsider this.

**Comment by sneridagh** [comment]
@tisto @davisagli I updated the description and reach of the PLIP.
